test/files/specialized/fft.scala

```diff
@@ -122,24 +123,27 @@ object Test  {
     else "input2.txt"
   }
 
-  def setUp: Unit = {
-//    print("Loading from %s.. ".format(inputFileName))
-    val f = io.Source.fromFile(inputFileName)
-    val lines = f.getLines
-    val n = lines.next.toInt
-    data = new Array[Complex](n)
-    var i = 0
-    for (line <- lines if line != "") {
-      val pair = line.trim.split(" ")
-      data(i) = (pair(0).trim.toDouble, pair(1).trim.toDouble)
-      i += 1
+  def setUp(): Unit = {
+    //print("Loading from %s.. ".format(inputFileName))
+    def readData() = Using(io.Source.fromFile(inputFileName)) { f =>
```

Comment:
Kept use of Source only because this was the first edit starting from the last output from grep.